exercises/octal/example.scm

```diff
@@ -0,0 +1,13 @@
+(define (to-decimal s)
+  (call/cc
+   (lambda (return)
+     (let loop ([cs (string->list s)]
+                [dec 0])
+       (cond
+        [(null? cs) dec]
+        [(and (char>=? (car cs) #\0)
+              (char<=? (car cs) #\7))
```

Comment:
Overall looks good. I just thought I'd mention a more stylish alternative to write this:
```scheme
(char<=? #\0 (car cs) #\7)
```

Reply:
:+1: 

Thanks!